### DISCUSSION TITLE
managing customers

### DISCUSSION THREAD

**Discussion by onno-timmerman** [plausible/analytics#1773] [Anything Else]
Hi,
I decided to move over to plausible. I only got one problem, its the billing. 
You can setup the sites very easy. But imagine the following case. I got 10 clients 5 small website, 3 mid websites en 2 large

How can I segment the billing for them. I need to bill something more to the larger once and smaller ones. I don't know in advance how large a website will get.

**Comment by metmarkosaric** [comment]
hi @onno-timmerman! 

one way would be to [transfer their sites](https://plausible.io/docs/transfer-ownership) to them after you've set them up if you want them to be responsible for the billing. 

other than that, it's really up to you how to do it. we charge you for the total number of pageviews between all the sites you have added so you could charge them according to the size of their page for instance. the first month of trial is a good time to figure out how large their site is. you can obviously change your charge at any time as your cost change or as their traffic changes
